i18n/ko.json

```diff
@@ -320,60 +320,60 @@
     "continueAnyway": "무시하고 계속",
     "directory": "디렉터리",
     "doesNotExist": "존재하지 않습니다.",
-    "HubAlreadyExists": "같은 이름의 Hub가 있음",
+    "hubAlreadyExists": "같은 이름의 Hub가 있음",
     "noSuchFileFound": "해당 파일을 찾을 수 없음:",
-    "pleaseChangeName": "위 Hub 이름을 변경하십시오.",
-    "selectDefaultPlayer": "선호하는 동영상 플레이어를 선택하세요.",
-    "selectPreviousHub": "이전에 저장한 동영상 Hub 파일을 선택하세요.",
+    "pleaseChangeName": "Hub 이름을 변경하세요.",
+    "selectDefaultPlayer": "기본으로 사용할 동영상 플레이어를 선택하세요.",
+    "selectPreviousHub": "이전에 저장된 동영상 Hub 파일을 선택하세요.",
     "selectRootFolder": "원본 동영상 폴더 선택",
     "videoFolderNotFound": "원본 동영상 폴더를 찾을 수 없음:"
   },
   "TAGS": {
-    "add": "Add",
+    "add": "추가",
     "addSuccess": "추가됨",
-    "addTag": "Add tag",
+    "addTag": "태그 추가",
     "alreadyExists": "태그가 이미 있음",
-    "any": "any",
-    "closeTagTray": "태그 트레이 닫기",
-    "delete": "delete mode",
-    "filterList": "Containing",
+    "any": "입력",
+    "closeTagTray": "태그 창 닫기",
+    "delete": "삭제 모드",
+    "filterList": "필터",
     "filterTags": "필터 태그",
-    "hidingFrequency": "빈도 숨기기",
-    "minimum": "minimum",
+    "hidingFrequency": "태그 개수 숨기기",
+    "minimum": "이상 포함된 태그",
     "presentTags": "이 Hub의 태그",
-    "showingFrequency": "빈도 보이기",
-    "showingMostSimilar": "비슷한 영상:",
+    "showingFrequency": "태그 개수 보이기",
+    "showingMostSimilar": "비슷한 동영상:",
     "sortAlphabetical": "이름순 정렬",
-    "sortByFrequency": "빈도로 정렬",
-    "starRating": "별점 등급",
+    "sortByFrequency": "태그 개수로 정렬",
+    "starRating": "별점 매기기",
     "timesPlayed": "재생 횟수"
   },
   "WIZARD": {
     "change": "변경",
     "clearRecent": "최근의 모든 기록 지우기",
-    "clipSnippets": "조각으로",
+    "clipSnippets": "개의 자투리 생성",
     "closeButton": "마법사 닫기",
     "createVideoHub": "동영상 Hub 만들기",
-    "demoVersion": "데모 버전은 50개 동영상으로 제한",
-    "eachSnippet": "각각의 조각들을",
-    "expected": "Expected:",
-    "extractClips": "클립 미리 보기 생성",
-    "found": "found",
-    "HubDirectory": "(Hub 디렉터리)",
-    "imported": "imported",
-    "importingFrom": "다음 위치에서 가져오기:",
+    "demoVersion": "50개 동영상으로 제한된 데모 버전",
+    "eachSnippet": "각각의 자투리를",
+    "expected": "예상:",
+    "extractClips": "",
```

Comment:
354 + 360 -> snippets Generate clip preview with 
360 Is there no problem even if it's blank?